### PR TITLE
Updates to Neurology journal format

### PR DESCRIPTION
**Pull request by bbbruce**
These changes remove issue numbers, remove month and day from the journal article type, correct online article support, and add e-pub support according to the journal style.

### CODE REVIEW

File: neurology.csl [hunk 1]
```diff
@@ -11,13 +11,15 @@
...
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="en">
     <terms>
-      <term name="retrieved">available</term>
+      <term name="retrieved">accessed</term>
+      <term name="from">at</term>
       <term name="section" form="short">sect.</term>
+      <term name="internet">online</term>
     </terms>
```
Comment (adam3smith) [inline neurology.csl]: at, accessed, and online all have terms of their own. Use those rather than redifining other ones, please. This is too confusing.

**Comment by adam3smith** [comment]
Otherwise this looks good, thanks! See instructions above on how to update your pull request.

**Comment by bbbruce** [comment]
Thanks for your suggestions.  It helped get rid of the cruft.

**Comment by adam3smith** [comment]
perfect, thanks!

**Pull request merged**